### PR TITLE
aacgain: Add package.

### PR DESCRIPTION
**Pull request by robbinch**
This adds aacgain which is a normalizer for AAC (.m4a) audio files.

### CODE REVIEW

File: pkgs/applications/audio/aacgain/default.nix [hunk 1]
```diff
@@ -0,0 +1,72 @@
+{ stdenv, fetchgit, unzip, m4, libtool, automake, autoconf, mp3gain, mp4v2, faad2 }:
+
+stdenv.mkDerivation {
+  name = "aacgain-1.9";
+  srcs = [
+    (fetchgit {
```
Comment (pSub) [inline pkgs/applications/audio/aacgain/default.nix]: Please use fetchFromGitHub, with the release tag as `rev`, that avoids cloning the full repository.

File: pkgs/applications/audio/aacgain/default.nix [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    cd $sourceRoot/mp4v2
+    ./configure
+
+    cd $sourceRoot/faad2
+    ./configure
+  '';
+
+  buildPhase = ''
+    cd $sourceRoot/mp4v2
+    make libmp4v2.la
```
Comment (pSub) [inline pkgs/applications/audio/aacgain/default.nix]: Do you really need to build those dependencies here?

**Comment by robbinch** [comment]
Fixed to use fetchFromGitHub.

@pSub Unfortunately, yes. The steps above follows upstream instructions on building for Linux, which is added as an afterthought to a Windows program, therefore building it is a bit unconventional.

**Comment by garbas** [comment]
merged [here](http://git.io/vI5av)

**Pull request closed without merging**